doc/Support/Configuration.md

```diff
@@ -100,6 +100,15 @@ $config['icmp_check'] = false;
 
 If you would like to do this on a per device basis then you can do so under Device -> Edit -> Misc -> Disable ICMP Test? On
 
+#### traceroute
+LibreNMS uses traceroute / traceroute6 to record debug information when a device is down due to icmp AND you have `$config['debug']['run_trace] = true;`
```

Comment:
Missing single quote after run_trace.

Reply:
Can you send a pull request to fix?

Reply:
Sure.  I'm new to this, but I should be able to figure it out and get that going.

Reply:
Easy way go to the docs page and click the edit button ;)

Reply:
Thanks.  I'll get this figured out eventually :)   Anyway, looks like someone else corrected it.